README.md

```diff
@@ -1 +1,47 @@
-# Awesomebooks
\ No newline at end of file
+![Hireable](https://cdn.rawgit.com/hiendv/hireable/master/styles/default/yes.svg)
+
+  <h1 align="center">Awesome Books</h1>
+
+### Built with
+
+- HTML
+- JavaScript(ES6)
+- Bootstrap 5
+- Browser local storage
+
+## Setup and run the app
+
+- Run `git clone https://github.com/muhenge/Awesomebooks.git` in your termianl
+- Run `npm install` from the terminal to install the dependences
```

Comment:
Good job adding the README file but let's update the file for non-technical users so that they can easily run your app on their local machine.

Kindly check out  the following steps:
Example: 
- git clone `your repo` ✅ 
- cd into the folder 
- open `index.html` with live server

Reply:
* As pointed out by the previous TSE you can still make the readme better by following the examples of the reviewer 
* in this project running `npm install` isn't necessary kindly put in the steps one would take to setup your project locally 👍

Reply:
I would like to ask if these are requirements to have the project approved and if they are I'd like to know why, for the sake of clarity, aren't they listed as requirements on the code review. Thank you.

Reply:
@webdevmaxi, Perhaps you have come across the [5 Basic Rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), in there you will number 5 with "Please use [this template](https://github.com/microverseinc/readme-template).

So in the template, you will see the `Setup` session.

I hope this is clear now.


### [Optional]
Kindly add a screenshot to your README file.
I marked this as optional because I forgot to add it in the first review.
It would be nice if you add it.

Reply:
Alright. I'll add a screenshot. I've removed the file as asked and updated the instructions in the read me file. Am I good to request a new code review?